recipes/exchange-calendars/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file is already included in the source distribution, you don't need to include it again.